inst/python_cmd.m

```diff
@@ -116,7 +116,7 @@
 
   % Careful: fix this constant if you change the code below.
   % Test with "python_cmd('raise')" which should say "line 1".
-  LinesBeforeCmdBlock = 3;
+  LinesBeforeCmdBlock = 4;
```

Comment:
leave as `4` as that is more correct (the number of lines of boilerplate in `cmd`).

Then use `A{3} + 1 - ...` below?